docs/api.json

```diff
@@ -0,0 +1,1070 @@
+          "intendedFor": ["twineAdmin", "fundingBody", "cbAdmin"],
+          "scope": ["user_details-own:delete"],
+          "body": {
+            "userName": "String"
+          },
+          "response": null
+        }
+      },
+      "/volunteers/me": {
+        "GET": {
+          "description": "Retrieve details of your own volunteer user account",
+          "auth": true,
+          "intendedFor": ["volunteer"],
+          "scope": ["user_details-own:read"],
+          "query": {
+            "fields": ["String"]
+          },
+          "response": "types.volunteerUser"
+        },
+        "PUT": {
+        "description": "Update details of a your own volunteer user account",
+        "auth": true,
+        "intendedFor": ["volunteer"],
+        "scope": ["user_details-own:write"],
+        "query": {
+          "name": "String",
+          "regionId": "Number",
+          "sectorId": "Number",
+          "organisationId": "Number",
+          "gender": "String",
+          "birthYear": "String",
+          "email": "String",
+          "phoneNumber": "String",
+          "emailConsent": "Boolean",
+          "smsConsent": "Boolean",
+          "delete": "Boolean"
+        },
+        "response": "types.volunteerUser"
+        },
+        "DELETE": {
+          "description": "Delete your own volunteer user account",
+          "auth": true,
+          "intendedFor": ["volunteer"],
+          "scope": ["user_details-own:delete"],
+          "body": {
+            "userName": "String"
+          },
+          "response": null
+        }
+      },
+      "/visitors/me": {
+        "GET": {
+          "description": "Retrieve details of your own user account",
+          "auth": true,
+          "intendedFor": ["visitor"],
+          "scope": ["user_details-own:read"],
+          "query": {
+            "fields": ["String"]
+          },
+          "response": "types.visitorUser"
+        },
+        "PUT": {
+        "description": "Update details of a your own visitor user account",
+        "auth": true,
+        "intendedFor": ["visitor"],
+        "scope": ["user_details-own:write"],
+        "query": {
+          "name": "String",
+          "regionId": "Number",
+          "sectorId": "Number",
+          "organisationId": "Number",
+          "gender": "String",
+          "birthYear": "String",
+          "email": "String",
+          "phoneNumber": "String",
+          "emailConsent": "Boolean",
+          "smsConsent": "Boolean",
+          "qrCode": "String",
+          "delete": "Boolean"
+        },
+        "response": "types.visitorUser"
+        },
+        "DELETE": {
+          "description": "Delete your own visitor user account",
+          "auth": true,
+          "intendedFor": ["visitor"],
+          "scope": ["user_details-own:delete"],
+          "body": {
+            "userName": "String"
+          },
+          "response": null
+        }
+      },
+      "/:id/visit_activities/:id": {
+      "POST": {
```

Comment:
1. Indentation is not right
2. The RESTful convention is to make additions to collections using a `POST` to the endpoint describing the _entire_ collection. That means the appropriate way to create a visit should be `POST /visits` or `POST /organisations/:id/visits`. The current `/users/:id/visit_activities/:id` refers to a single entity, not a collection, so `POST`ing to it breaks convention.